tests/presentation-definition/iso.org.18013.5.1.mdl.json

```diff
@@ -0,0 +1,67 @@
+{
+  "id": "mDL_presentation_request",
+  "input_descriptors": [
+    {
+      "id": "mDL_credential",
+      "name": "Mobile Driving License",
+      "purpose": "To verify your driving privileges and age.",
+      "constraints": {
+        "fields": [
+          {
+            "path": ["$.type"],
+            "filter": {
+              "type": "array",
+              "contains": {
+                "const": "iso.org.18013.5.1.mDL"
+              }
+            }
+          },
```

Comment:
This is not a valid mDL presentation definition, so I don't think we should add this to the repo as a test.